tests/Mapper/DoctrineCollectorTest.php

```diff
@@ -46,7 +47,7 @@ public function testClear(): void
         $collector = DoctrineCollector::getInstance();
         $collector->addIndex(stdClass::class, 'name', ['column']);
         $collector->addUnique(stdClass::class, 'name', ['column']);
-        $collector->addInheritanceType(stdClass::class, 'type');
+        $collector->addInheritanceType(stdClass::class, ClassMetadataInfo::INHERITANCE_TYPE_SINGLE_TABLE);
```

Comment:
I would do this change in a separate PR

Reply:
Done

Reply:
This change is necessary because:

1x: Passing other type than int as argument 2 for method Sonata\Doctrine\Mapper\DoctrineCollector::addInheritanceType() is deprecated since sonata-project/doctrine-extensions 1.x. It will accept only int in version 2.0.